contracts/stellar-multicall/src/types.rs

```diff
@@ -4,6 +4,7 @@ use soroban_sdk::{contracttype, Address, Symbol, Val, Vec};
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct FunctionCall {
     pub contract: Address,
+    pub approver: Address,
```

Comment:
```suggestion
    /// There must be an address authorizing each function call, even if the function call itself doesn't need authorization, to prevent frontrunning of individual calls within a multicall
    pub approver: Address,
```